.github/workflows/test.yml

```diff
@@ -30,6 +35,7 @@ jobs:
         name: vmtest
         with:
           kernel: ${{ matrix.kernel }}
+          pahole: next
```

Comment:
My fixes for btf_dump were only in next. I thought that since we already use latest-greatest llvm and kernel, doing the same for pahole won't hurt. But in the meantime my patches propagated to master, so let's drop this change.